build-api/api.go

```diff
@@ -11,12 +11,13 @@ import (
 )
 
 type DockerInfo struct {
-	BRANCH     string `json:"git_branch" binding:"required"`
+	DEPLOY_KEY string `json:"deploy_key" binding:"required"`
+	BRANCH     string `json:"branch" binding:"required"`
+	TAG        string `json:"tag"`
 	HASH       string `json:"git_hash" binding:"required"`
```

Comment:
branch won't be required moving forward. Builder will build off of either the tag OR the branch/hash combo.